### DISCUSSION TITLE
[Question][Enviornment Setting] Password of docker image

### DISCUSSION THREAD

**Discussion by gayeun4195** [prosyslab-classroom/cs348-information-security#658] [Q&A]
Name: Gaeun Noh

Hello, I'm trying to set the environment using docker + vs code, and I'm following the steps in https://gitlab.com/-/snippets/2506127.

I tried to install vi in my docker machine, which needs sudo permission. However, I don't know the password of the given docker image.

Can I know the password of my docker-image, prosyslab/classroom?

**Comment by tanapthetimid** [comment]
It's 1234

**Comment by KAIST-JongchanPark** [comment]
Probably, initial sudo password is 1234.

**Comment by 727yubin** [comment]
By the way, unless you keep your docker running indefinitely, installing any additional packages won't help, since they will be deleted once the container is closed.

**Comment by tanapthetimid** [comment]
> By the way, unless you keep your docker running indefinitely, installing any additional packages won't help, since they will be deleted once the container is closed.

You can restart a stopped container in docker without losing any data. Use `docker run` to run a new container and `docker start` to start a stopped container.
